projects/arlas-toolkit/src/lib/components/aias-download/aias-download.component.html

```diff
@@ -11,15 +11,17 @@
       <div class="step">
         <form [formGroup]="formGroup" class="form">
           <section class="field">
-            <div>
-              <span class="checkbox-hint">{{ 'Uncheck to download the image only' | translate }}</span>
-            </div>
-            <mat-checkbox [formControlName]="'raw_archive'">{{ 'Download the entire archive' | translate }}
-            </mat-checkbox>
-            <div class="warning-text">
-              <mat-icon>warning</mat-icon>
-              <span>{{ 'No crop on AOI will be taken into account' | translate }}</span>
-            </div>
+            <mat-form-field class="field" appearance="outline">
+              <mat-label>{{ 'Download type' | translate }}</mat-label>
+              <mat-select [formControlName]="'raw_archive'" class="data-cy-target_format">
+                <mat-option [value]="true">{{ ((data.nbProducts >= 2) ? 'Download archives' : 'Download archive') | translate }}</mat-option>
```

Comment:
If we want to experiment with some angular features, there is the plural pipe that could be useful to do that, I don't think we already used it in our code. However, if we want the keys to be properly extracted, I think you need to put the translate pipe right after the key, but you should test if that is the case first

https://v14.angular.io/api/common/I18nPluralPipe